setup.py

```diff
@@ -25,7 +25,7 @@
 version = "2.0.0-dev1"
 release_status = "Development Status :: 5 - Production/Stable"
 dependencies = [
-    "google-api-core[grpc] >= 1.21.0, < 2.0.0dev",
+    "google-api-core[grpc] >= 1.22.1, < 2.0.0dev",
```

Comment:
The `1.22.1` release is odd to have as a minimum:  it changes only docstrings.  Seems like we would want `1.22.0` (to pick up the feature, "allow quota project to be passed to create_channel"), or else `1.22.2` (bugfix, "only add quota project id if supported").